File: src/com/prey/PreyAccountData.java
/*******************************************************************************
 * Created by Carlos Yaconi
 * Copyright 2012 Fork Ltd. All rights reserved.
 * License: GPLv3
 * Full license at "/LICENSE"
 ******************************************************************************/
package com.prey;

public class PreyAccountData {

	private String apiKey;
	private String deviceId;
	private String name;
	//private String login;
	private String password;
	private String email;
	private String refererId;
	private String preyVersion;
	private boolean missing;

	public String getApiKey() {
		return apiKey;
	}

	public void setApiKey(String apiKey) {
		this.apiKey = apiKey;
	}

	public String getDeviceId() {
		return deviceId;
	}

	public void setDeviceId(String deviceId) {
		this.deviceId = deviceId;
	}

//	public String getLogin() {
//		return login;
//	}
//
//	public void setLogin(String login) {
//		this.login = login;
//	}

	public String getPassword() {
		return password;
	}

	public void setPassword(String password) {
		this.password = password;
	}

	public String getEmail() {
		return email;
	}

	public void setEmail(String email) {
		this.email = email;
	}

	public String getRefererId() {
		return refererId;
	}

	public void setRefererId(String refererId) {
		this.refererId = refererId;
	}

	public String getName() {
		return name;
	}

	public void setName(String name) {
		this.name = name;
	}

	public String getPreyVersion() {
		return preyVersion;
	}

	public void setPreyVersion(String preyVersion) {
		this.preyVersion = preyVersion;
	}

	public void setMissing(boolean missing) {
		this.missing = missing;
	}

	public boolean isMissing() {
		return missing;
	}

}

File: app/src/main/java/com/prey/kotlin/PreyAccountData.kt
/*******************************************************************************
 * Created by Orlando Aliaga
 * Copyright 2025 Prey Inc. All rights reserved.
 * License: GPLv3
 * Full license at "/LICENSE"
 ******************************************************************************/
package com.prey.kotlin

class PreyAccountData {
    private var apiKey: String? = null
    private var deviceId: String? = null
    private var name: String? = null
    private var password: String? = null
    private var email: String ? = null
    private var refererId: String? = null
    private var preyVersion: String? = null
    private var isMissing: Boolean = false

    fun getApiKey(): String {
        return apiKey!!
    }

    fun setApiKey(apiKey: String?) {
        this.apiKey = apiKey
    }

    fun getDeviceId(): String {
        return deviceId!!
    }

    fun setDeviceId(deviceId: String?) {
        this.deviceId = deviceId
    }

    fun getPassword(): String {
        return password!!
    }

    fun setPassword(password: String?) {
        this.password = password
    }

    fun getEmail(): String {
        return email!!
    }

    fun setEmail(email: String?) {
        this.email = email
    }

    fun getRefererId(): String {
        return refererId!!
    }

    fun setRefererId(refererId: String?) {
        this.refererId = refererId
    }

    fun getName(): String {
        return name!!
    }

    fun setName(name: String?) {
        this.name = name
    }

    fun getPreyVersion(): String {
        return preyVersion!!
    }

    fun setPreyVersion(preyVersion: String?) {
        this.preyVersion = preyVersion
    }

    fun setMissing(missing: Boolean) {
        this.isMissing = missing
    }

    fun isMissing(): Boolean {
        return isMissing
    }
}